include/bluetooth/mesh/scheduler_srv.rst

```diff
@@ -143,8 +143,8 @@ Each entry has the following fields:
 Extended models
 ***************
 
-The Scheduler Server extends the :ref:`bt_mesh_scene_srv_readme`.
-In addition, the Scheduler Setup Server extends the Scene Setup Server (see the :ref:`bt_mesh_scene_srv_readme` documentation) and the Generic Power OnOff Setup Server (see the :ref:`bt_mesh_ponoff_srv_readme` documentation).
+The Scheduler Server is a root model and a main model that does not extend any other model.
```

Comment:
```suggestion
The Scheduler Server is a root model and a main model that does not extend any other models.
```
I see that this sentence is coming from the model spec, so I won't change it :)
But I find it strangely formulated.
Isn't the entire definition of being a root model that it does not extend any other models? It feels like a repetition.
That said, I see that the newer revision of the spec does not explain what a root model is, unless there is a more updated version out there than the one I have from August last year... @omkar3141?
Maybe mentioning the root model isn't even necessary then, if we say "main model that does not extend any other models"?

Reply:
Simplified phrase